src/index.js

```diff
@@ -39,7 +39,26 @@ const create = (privateKey, value, seq, lifetime, callback) => {
   // Validity in ISOString with nanoseconds precision and validity type EOL
   const isoValidity = nanoDateEol.toISOStringFull()
   const validityType = ipnsEntryProto.ValidityType.EOL
+  _create(privateKey, value, seq, isoValidity, validityType, callback)
+}
+
+/**
+ * Same as create(), but prevent generating a Date().
```

Comment:
`Same as create(), but instead of generating a new Date, it receives the intended expiration time`